### DISCUSSION TITLE
font and graph too small from pathways, pfamDomains, plotOncodrive and drugs

### DISCUSSION THREAD

**Discussion by complexgenome** [PoisonAlien/maftools#1024] [Q&A]
Hi developer,
Thank you for this wonderful library. It's extremely good and helpful. I've been able to use and load data, tinker around using various library functions .
I'd like to create some visualizations: pathways, pfamDomains and plotOncodrive

I read data, get summary and pass through oncodrive step. The plots generated thereafter are too small in font, are crumpled and unreadable.

Code is as below:

```
mm1000_oncodrive<-maftools::oncodrive(maf=mafs_read,AACol='Protein_Change',minMut=5,pvalMethod='zscore')

maftools::plotOncodrive(res = mm1000_oncodrive, fdrCutOff = 0.1, useFraction = TRUE, labelSize = 0.5)

mm100.pfam = pfamDomains(maf = mafs_read, AACol = 'Protein_Change', top = 10)

dgi = drugInteractions(maf = mafs_read, fontSize = 0.75)

pathways(maf = mafs_read, plotType = 'treemap')
```

Please see attached graphs: plotonco, pfam, pathways and drugs. 

![oncodrive](https://github.com/PoisonAlien/maftools/assets/5683969/3a5ccb61-162d-4041-ab18-4b79bd938d3c)
![pfam](https://github.com/PoisonAlien/maftools/assets/5683969/c02f0255-1ca8-42e9-9ec4-7ed639516d84)
![pathways](https://github.com/PoisonAlien/maftools/assets/5683969/35c4c259-3652-4069-8dec-a1350cab0da9)
![drug](https://github.com/PoisonAlien/maftools/assets/5683969/8a173e13-f008-43a0-ace1-bb3fa3f8e6d6)

How do I improve this?

**Comment by PoisonAlien** [comment]
Hi,

I would recommend cleaning your prior plotting window with `dev.off()` before plotting a new one.  Also, you should save your plots to a pdf with sufficient width and height.
